cli/tauri-bundler/src/bundle/appimage_bundle.rs

```diff
@@ -74,7 +74,7 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<PathBuf>> {
 
   // create the shell script file in the target/ folder.
   let sh_file = output_path.join("build_appimage.sh");
-  common::print_bundling(format!("{:?}", &appimage_path).as_str())?;
```

Comment:
why did you change this from the full path to the filename? Every other place is still the full path.

Reply:
The `.deb` bundle log shows just the file name. The full path is displayed after anyways.